NEWS.md

```diff
@@ -3,6 +3,12 @@
 ## What's changed
 
 - New string method `to_titlecase()` (#371).
+- Subnamespace name "arr" as in  `<Expr>$arr$` & `<Series>$arr$` is deprecated in favor of "list". 
+  Finally at polars 0.9.0 the "arr" will be removed (#375).
```

Comment:
I don't know where this came from.

Reply:
I must have missed to move it to unrelated churn PR. It is in main now, will remove these lines from this PR